src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -379,6 +379,13 @@ protected void getChanges2(ChangeEventSourceContext context,
                                 .getResp()
                                 .getCdcSdkProtoRecordsList()) {
                             CdcService.RowMessage m = record.getRowMessage();
+
```

Comment:
cc @adithya-yb Is it better if CDC Service in the database sends a SafePoint only if a flag is set?